app/helpers/eq.js

```diff
@@ -1,7 +0,0 @@
-import { helper } from '@ember/component/helper';
```

Comment:
Remove this helper as I've now introduced [ember-truth-helpers](https://github.com/jmurphyau/ember-truth-helpers/)